.snippets/text/smart-contracts/polkaVM-warning.md

```diff
@@ -0,0 +1,2 @@
+!!! smartcontract "PolkaVM"
+    PolkaVM smart contracts with Ethereum compatibility are an **alpha release**.
```

Comment:
```suggestion
    PolkaVM smart contracts with Ethereum compatibility are in **early-stage development and may be unstable or incomplete**.
```

I feel like we need to stress that things might not work as expected... feel free to tweak it 🙂